projects/ad9695_fmc/common/ad9695_fmc_bd.tcl

```diff
@@ -0,0 +1,161 @@
+
+# RX parameters
+set RX_NUM_OF_LANES $ad_project_params(RX_JESD_L)           ; # L
+set RX_NUM_OF_CONVERTERS $ad_project_params(RX_JESD_M)      ; # M
+set RX_SAMPLES_PER_FRAME $ad_project_params(RX_JESD_S)      ; # S
+set RX_SAMPLE_WIDTH 16                                      ; # N/NP
+set RX_SAMPLES_PER_CHANNEL [expr $RX_NUM_OF_LANES * 32 / ($RX_NUM_OF_CONVERTERS * $RX_SAMPLE_WIDTH)]
+
+set adc_data_width [expr $RX_SAMPLE_WIDTH * $RX_NUM_OF_CONVERTERS * $RX_SAMPLES_PER_CHANNEL]
+set adc_dma_data_width 128
```

Comment:
adc_dma_data_width needs to be calculated with the same formula as adc_data_width